packages/integration-tests/src/reactiveHints.test.ts

```diff
@@ -93,6 +93,26 @@ describe("reactiveHints", () => {
     ]);
   });
 
+  it("can do read-only string hint", () => {
```

Comment:
These are some extra tests I happened to have locally, unrelated to the current change